README.md

```diff
@@ -54,9 +52,10 @@ A shortcut object represents a single action that can be performed in a client s
 | label                 | string   | The suggested text to display on the link                                                                                                                                                                                                                                                                                            |
 | uri                   | string   | URI pointing to the destination of the shortcut                                                                                                                                                                                                                                                                                      |
+| type                  | string   | (Optional) The type of token this shortcut applies to. Possible values are `fungible` or `collectible`. If omitted, the token will be assumed to be a `collectible`.                                                                                                                                                                 |
 | icon                  | string   | (Optional) The suggested icon to display on the link. The following options are available: `vote`, `vote-2`, `stake`, `stake-2`, `view`, `chat`, `tip`, `mint`, `mint-2`, `discord`, `twitter`, `x`, `instagram`, `telegram`, `leaderboard`, `gaming`, `gaming-2`, `generic-link`, `generic-add`                                     |
 | prefersExternalTarget | boolean  | (Optional) Whether the shortcut prefers to be opened outside of the client (e.g. Outside of Phantom's in-app browser). Defaults to `false`                                                                                                                                                                                           |
-| preferredPresentation | string   | (Optional) How the shortcut prefers to be displayed. Possible values are `default` and `immerse`. The platform would choose how that translates to their UX. Defaults to `immerse`                                                                                                                                                      |
+| preferredPresentation | string   | (Optional) How the shortcut prefers to be displayed. Possible values are `default` and `immerse`. The platform would choose how that translates to their UX. Defaults to `immerse`                                                                                                                                                   |
 | limitToCollections    | string[] | (Optional) A list of collection addresses that should display this shortcut. If provided, the client should only show the shortcut on collections that are in this array. Other collections that share the same `external_url` will not show this shortcut. Addresses should be provided as strings. Defaults to an empty array `[]` |
```

Comment:
In v2 `limitToCollections` only applies to `"type": "collectible"` and the analogous property for `"type": "fungible"` is `limitToTokenAddresses`